Dockerfile

```diff
@@ -9,16 +9,20 @@ RUN mkdir /var/run/sshd
 RUN echo "REPREPRO_BASE_DIR=/data/debian" > /etc/environment
 
 # Configure an reprepro user (admin)
-RUN adduser --system --group --shell /bin/bash --uid 600 --disabled-password --no-create-home reprepro
+RUN adduser --system --group --shell /bin/bash --uid 600 --disabled-password --home=/data reprepro
 
 # Configure an apt user (read only)
-RUN adduser --system --group --shell /bin/bash --uid 601 --disabled-password --no-create-home apt
+RUN adduser --system --group --shell /bin/bash --uid 601 --disabled-password --home=/data apt
 
 ADD sshd_config /sshd_config
 ADD run.sh /run.sh
 RUN chmod +x /run.sh
 
 ENV REPREPRO_DEFAULT_NAME Reprepro
+ENV REPREPRO_ALLOW wheezy>wheezy/dev
+ENV REPREPRO_CODENAME wheezy
+ENV REPREPRO_CODENAME_DEV wheezy/dev
+ENV REPREPRO_CODENAME_PROD wheezy/prod
```

Comment:
I'm not sure about this change because it's still quite limited (for example we still cannot use any number of distributions/architectures we want) and if one is not happy with the default configuration, he can already use its customized reprepro configuration using a docker volume.
I would accept a pull request that would allow more exhaustive configuration though (be able to use one or many custom distributions/architectures...)